Documentation/API/nft_the_world.yml

```diff
@@ -0,0 +1,513 @@
+swagger: "2.0"
+info:
+  description: "This is the API documentation for the NFT exchanging site"
+  version: "0.1.0"
+  title: "NFT the World!"
+  termsOfService: "http://localhost/"
+  contact:
+    url: "https://github.com/amosproj/amos2021ws07-nft-development"
+  license:
+    name: "MIT"
+    url: "https://spdx.org/licenses/MIT.html"
+host: "localhost"
+basePath: "/api/v1"
+schemes:
+- "https"
+paths:
+  /asset/{asset_name}:
+    get:
+      tags:
+        - "asset"
+      summary: "Browse item in category [asset_name]"
+      description: "Return a long list of items belonging to a queried category"
```

Comment:
This is cool but do we really feature NFT categories? At the moment we only offer only map region NFTs, or is it difficult to add later? Is Nürnburg and Riga supposed to be two different categories?